docs/elements/reference/sign-in.mdx

```diff
@@ -0,0 +1,289 @@
+<SignIn.Step name="verifications">
+  <SignIn.Strategy name="password">
+    <Clerk.Field name="password">
+      <Clerk.Label>password</Clerk.Label>
+      <Clerk.Input />
+      <Clerk.FieldError />
+    </Clerk.Field>
+    <SignIn.Action submit>Sign In</SignIn.Action>
+  </SignIn.Strategy>
+  <SignIn.Action navigate="choose-strategy">
+    Use another method
+  </SignIn.Action>
+</SignIn.Step>
+```
+
+### `<Step name="choose-strategy">`
+
+Allows a user to pick a new strategy to verify. This step can be rendered by navigating to `choose-strategy` using `<Action>`.
+
+#### Example
+
+```tsx filename="page.tsx"
+<SignIn.Step name="choose-strategy">
+  <SignIn.SupportedStrategy name="phone_code">
+    Send a code to your phone
+  </SignIn.SupportedStrategy>
+</SignIn.Step>
+```
+
+### `<Step name="forgot-password">`
+
+If the currently requested strategy is `password`, but a user can't remember their password, you can navigate them to the `forgot-password` step. They can begin the reset password flow or choose a new strategy. This step can be rendered by navigating to `forgot-password` using `<Action>`.
+
+#### Example
+
+```tsx filename="page.tsx"
+<SignIn.Step name="forgot-password">
+  <SignIn.SupportedStrategy name="reset_password_email_code">
+    Reset your password via Email
+  </SignIn.SupportedStrategy>
+  <p>or</p>
+  <SignIn.SupportedStrategy name="google">
+    Sign in with Google
+  </SignIn.SupportedStrategy>
+</SignIn.Step>
+```
+
+### `<Step name="reset-password">`
+
+If a user has requested a password reset and verified their identity, they will be navigated to `reset-password`. A password field should be rendered in this step to allow the user to set a new password.
+
+#### Example
+
+```tsx filename="page.tsx"
+<SignIn.Step name="reset-password">
+  <Clerk.Field name="password">
+    <Clerk.Label>New password</Clerk.Label>
+    <Clerk.Input />
+    <Clerk.FieldError />
+  </Clerk.Field>
+  <Clerk.Field name="confirmPassword">
+    <Clerk.Label>Confirm password</Clerk.Label>
+    <Clerk.Input />
+    <Clerk.FieldError />
+  </Clerk.Field>
+  <SignIn.Action submit>Update password</SignIn.Action>
+</SignIn.Step>
+```
+
+## `<Strategy>`
+
+Conditionally render its children depending on the authentication strategy that needs to be verified. Does not render any markup on its own.
+
+### Properties
+
+| Name | Type     | Description                                                       |
+| ---- | -------- | ----------------------------------------------------------------- |
+| `name` | `'saml' \| 'ticket' \| 'password' \| 'passkey' \| 'phone_code' \| 'email_code' \| 'web3_metamask_signature' \| 'reset_password_email_code' \| 'reset_password_phone_code' \| 'email_link' \| 'totp' \| 'backup_code' \| 'oauth' \| 'web3'` \| [OAuthStrategy](https://github.com/clerk/javascript/blob/956d8792fefe9d6a89022f1e938149b25503ec7f/packages/types/src/strategies.ts#L15) | The name of the strategy for which its children will be rendered. |
```

Comment:
we should wrap OAuthStrategy in a code snippet as well, we can wrap the entire line like this:

```suggestion
| `name` | <code>'saml' \| 'ticket' \| 'password' \| 'passkey' \| 'phone_code' \| 'email_code' \| 'web3_metamask_signature' \| 'reset_password_email_code' \| 'reset_password_phone_code' \| 'email_link' \| 'totp' \| 'backup_code' \| 'oauth' \| 'web3' \| [OAuthStrategy](https://github.com/clerk/javascript/blob/956d8792fefe9d6a89022f1e938149b25503ec7f/packages/types/src/strategies.ts#L15)</code> | The name of the strategy for which its children will be rendered. |
```

https://github.com/clerk/clerk-docs/assets/98043211/c425c15b-1450-48e6-a746-4f4312674899